instrumentation/ktor/ktor-2.0/javaagent/build.gradle.kts

```diff
@@ -9,11 +9,11 @@ muzzle {
   pass {
     group.set("org.jetbrains.kotlinx")
     module.set("ktor-server-core")
-    versions.set("[2.0.0,)")
+    versions.set("[2.3.12,)")
   }
 }
 
-val ktorVersion = "2.0.0"
+val ktorVersion = "2.3.12"
```

Comment:
Is this version bump necessary? For javaagent instrumentations we prefer to build/test agains the earliest supported version and test the latest version via the `-PtestLatestDeps=true` flag.

Reply:
No, it's not necessary, I just didn't know about that flag. Thanks, I will revert it